dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/TestCommon/CommonTestCode.cs

```diff
@@ -174,6 +180,29 @@ public static void GetSubtreeTest()
             Verify.IsNull(resourceCandidate);
         }
 
+        public static void TryGetSubtreeTest()
+        {
+            var resourceManager = new ResourceManager("resources.pri.standalone");
+            var resourceMap = resourceManager.MainResourceMap.TryGetSubtree("resources");
+            var resourceCandidate = resourceMap.GetValue("IDS_MANIFEST_MUSIC_APP_NAME");
+            var resource = resourceCandidate.ValueAsString;
+            Verify.AreEqual(resource, "Groove Music");
+
+            var ex = Verify.Throws<Exception>(() => resourceMap.GetValue("abc"));
+            Verify.AreEqual((uint)ex.HResult, 0x80073b17); // HRESULT_FROM_WIN32(ERROR_MRM_NAMED_RESOURCE_NOT_FOUND)
+            resourceCandidate = resourceMap.TryGetValue("abc");
+            Verify.IsNull(resourceCandidate);
```

Comment:
nit:
Empty line before each `GetValue`/`TryGetValue` call will be helpful for the reader's mind on the first visit (at least some minds like mine).
Same in test above and below.